target_info/target_info.lua

```diff
@@ -0,0 +1,80 @@
+local math = require('math')
+local string = require('string')
+local ui = require('ui')
+local target = require('target')
+
+local text_color = ui.color.grey
+local config_state = {
+    title = 'Target Info',
+    style = 'chromeless',
+    x = 100,
+    y = 0,
+    width = 130,
+    height = 48,
+    color = ui.color.black,
+}
+function round(num)
+    return math.floor((num + 0.5))
+end
+
+ui.display(function()
+    ui.window('target_info', config_state, function()
+        local entity = target.st or target.t
+        if entity then
+
+            local entity_speed = ((entity.status == 5 or entity.status == 85) and round(100 * (entity.movement_speed / 4))) or round(100 * (entity.movement_speed / 5 - 1))
+
+            if entity_speed > 0 then
+                text_color = ui.color.lightgreen
+            elseif entity_speed < 0 then
```

Comment:
Just for my understanding... how can speed be negative? Or is that deviation from the default speed? Hence the division by `4`/`5` above?

Reply:
Players got a speed increase. Mounts and Chocobos did not. that's why 5 vs 4

Things such as gravity can cause negative speed.
Monsters also did not get the speed increase and a lot are now at negative 20% because of this compared to a player.